content/docs/0.8.x/continuous_delivery/deployment_helm/index.md

```diff
@@ -75,6 +75,55 @@ carts-primary   1/1     1            1            4m   carts        docker.io/ke
 carts           0/0     0            0            1d   carts        docker.io/keptnexamples/carts:0.11.2   app=carts
 ```
 
+
+## User Managed deployments (experimental)
+
+*Note: This feature is currently marked as experimental. Hence, expect various enhancements and/or changes in
+future releases of Keptn.*
+
+
+If the deployment strategy of a stage in the [shipyard](https://github.com/keptn/examples/blob/0.8.0/onboarding-carts/shipyard.yaml)
+is configured as `deploymentstrategy: user_managed`, the provided helm chart will be deployed without any modification and
+applied as it is.
+
+Assuming that you've created a project from a shipyard file containing a stage with `deploymentstrategy: user_managed`,you need to: 
+
+(1) create a service in your stage:
+```
+keptn create service <service-name> --project <proejct-name>
+```
+(2) add your desired Helm chart to each stage:
+```
+keptn add-resource --project=<project-name> --service=<service-name> --all-stages --resource=<your-helm-chart.tgz> --resourceUri=helm/<service-name>.tgz
+```
+(3) send an event to trigger the delivery of your service:
+
+```
+keptn send event --file=./delivery.json 
+```
+where the content of `delvier.json` looks something like:
+
+```
+{
+  "contenttype": "application/json",
+  "data": {
+    "project": "<project-name",
+    "service": "<service-name>",
+    "stage": "<stage-name>" 
+  },
+  "source": "https://github.com/keptn/keptn/cli#configuration-change",
```

Comment:
Source should be something different. It's certainly not coming from the Keptn CLI, nor is it a configuration-change.
@johannes-b any idea?

Reply:
agree, what about `keptn-user` ? afaik the bridge does not really care (?)

Reply:
None of the components care, but for traceability reasons it makes sense to have something in there that shows where it request is coming from. IMHO `keptn-user` is good (or if we want to use the same wording as with quality-gates, `keptn-cli`) in there.